create_exitnode.sh

```diff
@@ -209,5 +209,7 @@ service babeld start
 
 systemctl start babeld-monitor.timer
 systemctl enable babeld-monitor.timer
+systemctl stop exitnode-sync.timer
+systemctl start exitnode-sync.timer
```

Comment:
Hmm. Do you need to do `systemctl enable` here? I think that might be needed to make the service enabled after reboot? I'm not positive though.

Reply:
yep, i put stop instead of start, and start should be enable. i updated the file. i updated the file.